tools/PI/TestTools/TestDumpAnalyzer/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace TestDumpAnalyzer;
+
+internal sealed class Program
+{
+    public static int Main(string[] args)
+    {
+        try
+        {
+            Console.WriteLine("args are: " + args.Length + " " + string.Join(" ", args));
```

Comment:
Just making sure, is this test tool intended to be part of the PR code change?

Reply:
Yep. It's not hooked up to be automated yet, and I didn't include it in DevHome's build, but it's there as a starting point.